src/main/java/ch/uzh/ifi/hase/soprafs22/lobby/Lobby.java

```diff
@@ -250,12 +254,62 @@ public Player generatePlayer() {
         return new Player(generatedId, username, token, team);
     }
 
+    private String generateUniqueNameInLobby(){
+        String playerName = "Player-";
```

Comment:
Just an idea, have a list of names, a list of adjective and generate the name as randomAdjective + randomName. No need to do it.